probe_py/probe_py/analysis.py

```diff
@@ -240,7 +250,9 @@ def get_last_pthread(pid: int, exid: int, target_pthread_id: int) -> list[Node]:
         elif isinstance(op_data, ExecOp):
             # Exec brings same pid, incremented exid, and main thread
             target = pid, exid + 1, pid
-            exec_edges.append((node, first(*target)))
+            targetNode = first(*target)
```

Comment:
There is a possibility that the value of the key target in the `proc_to_ops` dict is an empty array, since we ignore all the ops except the `proc_ops`, hence added a check for None before creating an edge